### PR TITLE
APIGOV-26666 - change api validator frequency to cron schedule

### PR DESCRIPTION
**Pull request by dgghinea**

### CODE REVIEW

**Review by sbolosan — DISMISSED** [review]
Looks good. We will need to update docs

File: pkg/config/centralconfig.go [hunk 1]
```diff
@@ -783,7 +784,7 @@ func AddCentralConfigProperties(props properties.Properties, agentType AgentType
 	props.AddStringProperty(pathProxyURL, "", "The Proxy URL to use for communication to Amplify Central")
 	props.AddDurationProperty(pathPollInterval, 60*time.Second, "The time interval at which the central will be polled for subscription processing")
 	props.AddDurationProperty(pathReportActivityFrequency, 5*time.Minute, "The time interval at which the agent polls for event changes for the periodic agent status updater")
-	props.AddDurationProperty(pathAPIValidationFrequency, 1*time.Hour, "The time interval at which the agent validates API Services with the dataplane", properties.WithLowerLimit(5*time.Minute), properties.WithQAOverride())
+	props.AddStringProperty(pathAPIValidationCronSchedule, "0 * * * *", "The cron schedule at which the agent validates API Services with the dataplane")
```
Comment (vivekschauhan) [inline pkg/config/centralconfig.go:787]: Too aggressive schedule, I guess we decided to use daily as default

**Review by vivekschauhan — APPROVED** [review]

**Review by alrosca — APPROVED** [review]

File: pkg/config/centralconfig.go [hunk 2]
```diff
@@ -783,7 +784,7 @@ func AddCentralConfigProperties(props properties.Properties, agentType AgentType
 	props.AddStringProperty(pathProxyURL, "", "The Proxy URL to use for communication to Amplify Central")
 	props.AddDurationProperty(pathPollInterval, 60*time.Second, "The time interval at which the central will be polled for subscription processing")
 	props.AddDurationProperty(pathReportActivityFrequency, 5*time.Minute, "The time interval at which the agent polls for event changes for the periodic agent status updater")
-	props.AddDurationProperty(pathAPIValidationFrequency, 1*time.Hour, "The time interval at which the agent validates API Services with the dataplane", properties.WithLowerLimit(5*time.Minute), properties.WithQAOverride())
+	props.AddStringProperty(pathAPIValidationCronSchedule, "0 0 * * *", "The cron schedule at which the agent validates API Services with the dataplane")
```
Comment (jcollins-axway) [inline pkg/config/centralconfig.go:787]: the default can simply be @daily which is equivalent to 0 0 * * *, its just easier to understand at a glance

Comment (dgghinea) [reply pkg/config/centralconfig.go:787]: oh. I actually didn't know you could do that. thanks!

Comment (jcollins-axway) [reply pkg/config/centralconfig.go:787]: We probably need to add a QA override and do some cron checks to make sure it is not configured for more often than our previous lower limit of 5 minutes.

Check out the cron schedule validation in the usage config, we essentially get the next number of x cron scheduled intervals and check that the difference between them is never lower than our limit.

**Review by vivekschauhan — APPROVED** [review]

**Review by jcollins-axway — APPROVED** [review]

**Pull request merged**